problems/2.functions.js

```diff
@@ -3,15 +3,15 @@
 * @returns {null}
 */
 function logGreeting() {
-
+  let name = "Hello Avery!"
 }
-
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
```

Comment:
Stop ignoring this line. These parameters are REQUIRED and will be needed to pass the tests. When your function is called arguments will be passed in.